spec/yaml/swiftnav/sbp/skylark.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Swift Navigation Inc.
+# Contact: https://support.swiftnav.com
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.skylark
+description: Messages from the Skylark corrections service.
+stable: False
+public: True
+include:
+  - types.yaml
+definitions:
+  - MSG_ACKNOWLEDGE:
+      id: 0x0FA0
+      short_desc: Acknowledgement message in response to a request for corrections
+      fields:
+        - request_id:
+            type: u8
+            desc: Echo of the request ID field from the corresponding CRA message, or 255 if no request ID was provided.
+        - area_id:
+            type: u32
+            desc: Echo of the Area ID field from the corresponding CRA message.
+        - response_code:
+            type: u8
+            desc: Reported status of the request.
+            fields:
+              - 0-7:
+                  desc: Response code
+                  values:
+                      - 0: Ok
+                      - 1: Out of coverage
+                      - 2: Forbidden
+                      - 3: Invalid request
+                      - 4: Invalid area id
+        - correction_mask_on_demand:
+            type: u16
+            desc: >
+              Contains the message group(s) that will be sent in response from the corresponding CRA correction mask.
+              An echo of the correction mask field from the corresponding CRA message.
```

Comment:
this can be a bit mask field.

<div dir="ltr" style="margin-left:-5pt;" align="left" id="docs-internal-guid-11cbcb9f-7fff-6bf6-2f1b-ece285a99814">

Name | Bit | Description
-- | -- | --
Key | 0 | Public Signing Key
Low-Rate Messages | 1 | ITRF, UTC and APC Messages
Ephemeris | 2 | Ephemeris data
Satellite Clock | 3 | Clocks
Satellite Orbit | 4 | Orbit
Satellite code bias | 5 | Code Biases
Satellite phase bias | 6 | Phase Biases
Atmospherics | 7 | Tile Definition, STEC and Grid data
Integrity | 8 | Bounds, Degradation, and Flags
Reserved | 9-15 |  

</div>

Reply:
Ahh right thanks, added it